### ISSUE TITLE
Mass Obsoletion: focus on branch 'endocrine system disorder' (MONDO:0005151)

### ISSUE THREAD

**Issue by sabrinatoro** [monarch-initiative/mondo#6651]
Branch: 'endocrine system disorder' (MONDO:0005151)

Terms to obsolete:

obsoletion candidate ID | obsoletion candidate name
-- | --
http://purl.obolibrary.org/obo/MONDO_0017961 | 46,XX disorder of gonadal development
http://purl.obolibrary.org/obo/MONDO_0020039 | 46,XX disorder of sex development induced by androgens excess
http://purl.obolibrary.org/obo/MONDO_0017963 | 46,XX disorder of sex development induced by endogenous maternal-derived androgen
http://purl.obolibrary.org/obo/MONDO_0017964 | 46,XX disorder of sex development induced by exogenous maternal-derived androgen
http://purl.obolibrary.org/obo/MONDO_0019593 | 46,XX disorder of sex development induced by fetal androgens excess
http://purl.obolibrary.org/obo/MONDO_0017962 | 46,XX disorder of sex development induced by fetoplacental androgens excess
http://purl.obolibrary.org/obo/MONDO_0019608 | 46,XX disorder of sex development induced by maternal-derived androgen
http://purl.obolibrary.org/obo/MONDO_0017966 | 46,XY disorder of gonadal development
http://purl.obolibrary.org/obo/MONDO_0019597 | 46,XY disorder of sex development due to isolated 17,20-lyase deficiency
http://purl.obolibrary.org/obo/MONDO_0017974 | 46,XY disorder of sex development induced by maternal-exposure to endocrine disruptors
http://purl.obolibrary.org/obo/MONDO_0017969 | 46,XY disorder of sex development of endocrine origin
http://purl.obolibrary.org/obo/MONDO_0015130 | acquired chronic primary adrenal insufficiency
http://purl.obolibrary.org/obo/MONDO_0017240 | acrodysostosis with multiple hormone resistance
http://purl.obolibrary.org/obo/MONDO_0018532 | adenocarcinoma of liver and intrahepatic biliary tract
http://purl.obolibrary.org/obo/MONDO_0015777 | adult hypothyroidism
http://purl.obolibrary.org/obo/MONDO_0019800 | chronic hepatic porphyria
http://purl.obolibrary.org/obo/MONDO_0017709 | disorder of lipid absorption and transport
http://purl.obolibrary.org/obo/MONDO_0017581 | familial infantile gigantism
http://purl.obolibrary.org/obo/MONDO_0028618 | gastroenteric neuroendocrine neoplasm
http://purl.obolibrary.org/obo/MONDO_0957024 | hereditary 46,XX disorder of sex development
http://purl.obolibrary.org/obo/MONDO_0957025 | hereditary 46,XY disorder of sex development
http://purl.obolibrary.org/obo/MONDO_0015509 | hereditary biliary tract disease
http://purl.obolibrary.org/obo/MONDO_0015508 | hereditary parenchymatous liver disease
http://purl.obolibrary.org/obo/MONDO_0019505 | hypomyelination-hypogonadotropic hypogonadism-hypodontia syndrome
http://purl.obolibrary.org/obo/MONDO_0025511 | inherited neuroendocrine tumor
http://purl.obolibrary.org/obo/MONDO_0018575 | microcephalic primordial dwarfism-insulin resistance syndrome
http://purl.obolibrary.org/obo/MONDO_0019758 | midline interhemispheric variant of holoprosencephaly
http://purl.obolibrary.org/obo/MONDO_0016721 | pineal tumor of neuroepithelial tissue
http://purl.obolibrary.org/obo/MONDO_0019844 | pituitary hormone deficiency secondary to storage disease
http://purl.obolibrary.org/obo/MONDO_0016409 | primary congenital hypothyroidism
http://purl.obolibrary.org/obo/MONDO_0020524 | primary parathyroid hyperplasia
http://purl.obolibrary.org/obo/MONDO_0034217 | resistance to thyroid hormone due to a mutation in thyroid hormone receptor beta
http://purl.obolibrary.org/obo/MONDO_0016355 | semilobar holoprosencephaly
http://purl.obolibrary.org/obo/MONDO_0017218 | septopreoptic holoprosencephaly
http://purl.obolibrary.org/obo/MONDO_0034661 | syndromic biliary atresia
http://purl.obolibrary.org/obo/MONDO_0016565 | syndromic genetic obesity
http://purl.obolibrary.org/obo/MONDO_0015778 | syndromic hypothyroidism

**Comment by nicolevasilevsky** [comment]
Reassigning @katiermullen as we discussed on the call last week. Thanks @katiermullen!

**Comment by nicolevasilevsky** [comment]
[spreadsheet is here](https://docs.google.com/spreadsheets/d/17m7z02utbMDhGzazrJJJFL98pzZfbXg_kPxi6RYLezE/edit#gid=289446069)

**Comment by katiermullen** [comment]
@nicolevasilevsky @twhetzel please review and comment on 'orphaned' and 'leaves the branch' designees in the spreadsheet. Thank you!

**Comment by nicolevasilevsky** [comment]
Review is done! Nice job @katiermullen 

@sabrinatoro do you want to take a look at my comments and make sure you agree?

**Comment by katiermullen** [comment]
I made some edits to this branch based on "syndromic disease" discussion #6768

**Comment by sabrinatoro** [comment]
I reviewed and it looks good. It is ready for PR. 
Thank you @katiermullen

**Comment by sabrinatoro** [comment]
Expert review feedback: https://docs.google.com/spreadsheets/d/1yJ08nxP3Eu59G3wwVO8KXc6gw6Mh21D2/edit#gid=547601471 


TO DO:
- [ ] add new parents ("ROBOT Reviewer-Curator Agree - Stay" Tab)
- [ ] add new comments ("ROBOT Reviewer-Curator Agree - Leave" Tab) !!!!1 term comment will need to be added manually
- [ ] add new parents ("ROBOT Reviewer-Stay Curator-Leave" Tab)
- [ ] review "Reviewer-Leave Curator-Stay" Tab [TO DO WILL BE UPDATED ACCORDINGLY]


Note: Once the to-do list is done, all terms will have been reviewed and this issue can be closed

**Comment by sabrinatoro** [comment]
There are some comments from the community that should be reviewed, but this is future work. This issue can be closed for now.
